nvd3/__init__.py

```diff
@@ -26,5 +27,5 @@
 from .scatterChart import scatterChart
 from .discreteBarChart import discreteBarChart
 from .multiBarChart import multiBarChart
-from .bulletChart import bulletChart
```

Comment:
You removed bulletChart? any reason?

Reply:
Sorry, a careless mistake, added back in.